tests/suites/smoke/test_api_bootstrap.py

```diff
@@ -0,0 +1,16 @@
+# project aquarium's testing battery
```

Comment:
It's totally a nit, but I'd personally like to avoid naming files in the unit-test format of `test_*.py` to avoid confusion with test runners. Perhaps `aqrtest_*.py`?